### PR TITLE
[Feature] Add FastFCN

### PR DESCRIPTION
**Pull request by justld**
添加模型 Fastfcn: Rethinking dilated convolution in the backbone for semantic segmentation

### CODE REVIEW

File: configs/fastfcn/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# FastFCN: Rethinking Dilated Convolution in the Backbone for Semantic Segmentation
+
+## Reference
+> Huikai Wu, Junge Zhang, Kaiqi Huang. "FastFCN: Rethinking Dilated Convolution in the Backbone for Semantic Segmentation".
```
Comment (shiyutang) [inline configs/fastfcn/README.md]: The reference is not complete, please change to : Wu, Huikai, Junge Zhang, Kaiqi Huang, Kongming Liang, and Yizhou Yu. "Fastfcn: Rethinking dilated convolution in the backbone for semantic segmentation." arXiv preprint arXiv:1903.11816 (2019).

File: configs/fastfcn/fastfcn_resnet50_os8_ade20k_480x480_120k.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  num_codes: 32
+  mid_channels: 512
+  use_jpu: True
+  aux_loss: True
+  use_se_loss: True
+  add_lateral: True
+
+loss:
+  types:
+    - type: CrossEntropyLoss # output loss
```
Comment (shiyutang) [inline configs/fastfcn/fastfcn_resnet50_os8_ade20k_480x480_120k.yml]: remove annotation plz.

File: paddleseg/models/fastfcn.py [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+            self.fusion = layers.ConvBNReLU(
+                3 * mid_channels,
+                mid_channels,
+                3,
+                padding=1,
+                bias_attr=False,
+            )
+
+        self.enc_module = EncModule(mid_channels, num_codes)
+        self.cls_seg = nn.Sequential(nn.Conv2D(mid_channels, num_classes, 1), )
```
Comment (shiyutang) [inline paddleseg/models/fastfcn.py]: nn.Sequential seems to be unnecessary.

**Review by shiyutang — CHANGES_REQUESTED** [review]

File: configs/fastfcn/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+## Performance
+
+### ADE20K
+
+| Model | Backbone | Resolution | Training Iters | mIoU | mIoU (flip) | mIoU (ms+flip) | Links |
+|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|
+|FastFCN|ResNet50_vd|480x480|120000|-|-|-|-|
```
Comment (shiyutang) [inline configs/fastfcn/README.md]: plz add the mIOU results, including flip and ms+flip

File: configs/fastfcn/fastfcn_resnet50_os8_ade20k_480x480_120k.yml (hunk 2 above)
Comment (justld) [reply configs/fastfcn/fastfcn_resnet50_os8_ade20k_480x480_120k.yml]: removed.

File: configs/fastfcn/README.md (hunk 1 above)
Comment (justld) [reply configs/fastfcn/README.md]: changed.

File: paddleseg/models/fastfcn.py (hunk 3 above)
Comment (justld) [reply paddleseg/models/fastfcn.py]: removed

File: configs/fastfcn/README.md (hunk 4 above)
Comment (justld) [reply configs/fastfcn/README.md]: added.

**Comment by shiyutang** [comment]
The test result of retrained model is live up to expectation.
image

**Review by shiyutang — DISMISSED** [review]
LGTM

**Review by shiyutang — APPROVED** [review]
LGTM after merge

**Pull request merged**